### DISCUSSION TITLE
Hi,

### DISCUSSION THREAD

**Discussion by ploverlover** [opengisch/QField#3926] [Q&A]
I'm a beta user trying to get my data usage below 100Mb cloud storage for the moment.
I've deleted test projects, images etc until I have 142Mb total storage, of which 300Kb is my actual files.
The balance appears to be change records some of which go back a long time.
Can I delete change records, and if so how can I go about it and manage the storage date to stay within the 100Mb limit

**Comment by ZsanettMed** [comment]
Hi @ploverlover 

You can delete old file versions: files->3 dots -> delete
<img width="773" alt="image" src="https://user-images.githubusercontent.com/83091224/220313847-197b3053-5798-48d4-9ef6-61ecb3342f7c.png">

**Reply by ploverlover** [reply]
Thanks - i get an error on trying to delete old versions_
Select versions older than one month - Server Error (500).
On navigating back to the original screen I find versions older than one month have not been deleted

**Reply by AnomalousExploration** [reply]
Is there a way to delete multiple versions all at once so only the current version is stored online? I'm worried that when multiple people are uploading data to the project every day the older versions will become a very tedious activity to delete. Thanks!

**Reply by ZsanettMed** [reply]
Thank you for your feedback @ploverlover , we could reproduce this issue, it will be fixed as soon as possible. You can delete older file versions one-by-one. 
@AnomalousExploration yes, it is possible, but currently not working.

**Comment by AnomalousExploration** [comment]
Thanks.

To confirm, there is no way currently to mass delete older versions or
automatically delete older versions of the project on qfieldcloud?

Thanks
Ellen

**Reply by ZsanettMed** [reply]
@ploverlover & @AnomalousExploration, Now, you can delete older versions again, the bug has been fixed. Apologizes for any inconvenience this may have caused.

Best regards, 
Zsanett

**Comment by AnomalousExploration** [comment]
Thank you Zsanett!

Is there a way to delete multiple versions at once. I'd love to see that,
if there isn't yet. or away to only save the current version and
automatically delete older versions after a 3 day period.
Thanks,
ellen

Anomalous Exploration

Grassroots Geological Services with a Lighter Footprint

www.anomalous.rocks
